plasma_framework/contracts/src/framework/registries/VaultRegistry.sol

```diff
@@ -24,6 +30,7 @@ contract VaultRegistry is Operated {
 
         _vaults[_vaultId] = _vaultAddress;
         _vaultToId[_vaultAddress] = _vaultId;
+        quarantine(_vaultAddress);
```

Comment:
I would write `Quarantine.quarantine(...)` to explicit where this function from to make code tracing easier.